crates/holochain/src/core/ribosome/host_fn/call.rs

```diff
@@ -147,19 +146,11 @@ pub fn call(
                                             nonce,
                                             expires_at,
                                         };
-                                        let call = ZomeCall::try_from_params(
```

Comment:
Bridge calls (calls to other cells on the same conductor) do not require authentication.